src/zarr/core/array.py

```diff
@@ -252,12 +252,6 @@ async def _create_v3(
         shape = parse_shapelike(shape)
         codecs = list(codecs) if codecs is not None else [BytesCodec()]
 
-        if fill_value is None:
```

Comment:
I don't understand this. What happens if I don't supply a fill_value for a uint8 array?

Reply:
for v3, that gets handled in `parse_fill_value`. That's why I removed it, didn't seem right to set a default in two places.

Reply:
Cool. Thanks for explaining!